client/src/components/reviews/ProductBreakdown.jsx

```diff
@@ -11,19 +11,17 @@ const ProductBreakdown = ({data}) => {
   const [listOfCharacteristics, setListOfCharacteristics] = useState(storeCharacteristicsInArray(characteristics));
 
   return (
-    <>
-      <div>
-        {listOfCharacteristics.map((value, index) => {
-          return (
-            <div key={value.details.id}>
-              <div className="product-breakdown" >
-                <Characteristic value={value}/>
-              </div>
+    <div>
```

Comment:
This is so much easier for me to parse rather than the fragments, a failing on my part I suppose.